docs/hardware/devices/rak/peripherals.mdx

```diff
@@ -29,7 +29,7 @@ Further information on the RAK1910 can be found on the [RAK Documentation Center
 
 US Distributor - Purchase link: [Rokland](https://store.rokland.com/products/rak-wireless-rak1910-wisblock-gnss-location-module)
 
-To add a GPS to the RAK19003 base board, you need the [RAK12500 GPS sensor](https://store.rakwireless.com/products/wisblock-gnss-location-module-rak12500). It is supported via I<sup>2</sup>C on slot B for firmware versions 1.49 and above.
+To add a GPS to the RAK19003 or RAK19007 base boards, you need the [RAK12500 GPS sensor](https://store.rakwireless.com/products/wisblock-gnss-location-module-rak12500). It is supported via I<sup>2</sup>C on slot B for firmware versions 1.49 and above.
```

Comment:
Hmm, the RAK1910 is also supported on the 19003 and 19007.  I can't remember off the top of my head what slots these use.

Reply:
Looking at the RAK docs, it looks like A as well, maybe. I realize my docs update is wrong anyway, since it says to use slot B. And I'm not sure that's right either, the RAK docs seem to suggest the 19003 has only slots C and D (and it looks like C is the one that's the right size for either GPS module), no A or B at all. So something's amiss, anyway.

Reply:
I updated it to say that you _should_ get the RAK12500 (since the RAK1910 on their site suggests the RAK12500 as an updated replacement anyway, that seems correct, and corrected my notes about which slot (A) to use on the RAK19007. I don't really know the correct answers for the other boards, so I haven't touched those.